coder_sniffer/Drupal/ruleset.xml

```diff
@@ -6,6 +6,7 @@
  <arg name="encoding" value="utf-8"/>
 
  <autoload>./coder_unique_autoload_phpcs_bug_2751.php</autoload>
+ <config name="installed_paths" value="../../slevomat/coding-standard,../../vendor/slevomat/coding-standard"/>
```

Comment:
I'm not sure about this. Ideally we'd be able to include sniffs using a fully qualified classname - if they are available to the autoloader and not have to mess with PHPCS configuration at all. For the unused variable analysis we added a class that extended the `sirbrillig/phpcs-variable-analysis` but that does not feel right.

Reply:
I would probably vote agianst adding the installed_paths. Although the documentation on how to add the required sniffs manually needs updating then. (https://www.drupal.org/docs/contributed-modules/code-review-module/installing-coder-sniffer#s-register-coder-standards)

Also, shouldn't the codesniffer autoload file take care of loading the sniffs automaticly? At least when `vendor/bin/phpcs` is used? (see https://github.com/squizlabs/PHP_CodeSniffer/search?q=autoload)

Reply:
Well the problem is that the tests fail without this line. The whole way standards are loaded and PHPCS uses global configuration is a PITA. Thinking about it more I think we need to open a PHPCS issue on the best way to extend other people's coding standards which is what we want to do here.

Reply:
Phpunit runs fine locally without since the bootstrap includes the autoload.php...? Or perhaps this was because I've got the global composer plugin. I'll have an extra look.

Reply:
I have no global packages. So im kinda confused why it would fail if we remove this. It really shouldn't.

Actually, it seems the composer plugin is actually installed with the new dependency, which in turn will mean the standard is loaded?

```
$ composer why dealerdirect/phpcodesniffer-composer-installer
slevomat/coding-standard  7.0.18  requires  dealerdirect/phpcodesniffer-composer-installer (^0.6.2 || ^0.7)
```

Reply:
@bbrala nice - this all works - I guess I wonder if we should add a direct dependency on dealerdirect/phpcodesniffer-composer-installer to coder?

Reply:
Yeah i think we should since this will make it explicit and it helps in using this package.